### PR TITLE
Updates serve modern code to include bugfixes property

### PR DESCRIPTION
**Pull request by housseindjirdeh**
Changes proposed in this pull request:

- Adds a section on using `bugfixes` property  with `@babel/preset-env`

CC @connorjclark @developit

### CODE REVIEW

File: src/site/content/en/fast/serve-modern-code-to-modern-browsers/index.md [hunk 1]
```diff
@@ -116,6 +108,35 @@ browsers to be used to access your application.
...
 target browsers that fit your needs.
 
+### @babel/preset-modules
+
+`@babel/preset-env` groups multiple JavaScript syntax features into collections and enables/disables
+them based on the target browsers specified. Although this works well, an entire collection of
+syntax features is transformed when a targeted browser contains a bug with just a single feature.
+This results in more transformed code than is necessary.
+
+A separate preset, [`@babel/preset-modules`](https://github.com/babel/preset-modules) solves this
```
Comment (connorjclark) [inline src/site/content/en/fast/serve-modern-code-to-modern-browsers/index.md:118]: If `bugfixes: true` and `preset-modules` are equivalent, should we even mention the latter? if they are equivalent today, will it remain that way or will the latter be more comprehensive?

File: src/site/content/en/fast/serve-modern-code-to-modern-browsers/index.md [hunk 2]
```diff
@@ -116,6 +108,35 @@ browsers to be used to access your application.
...
+problem by converting broken syntax to the closest non-broken, but still modern, syntax that would
+work for the targeted browsers. Instead of needing to install this preset separately, enable the
+[`bugfixes`](https://babeljs.io/docs/en/babel-preset-env#bugfixes) property to add these
+optimizations directly to `@babel/preset-env`.
```
Comment (developit) [inline src/site/content/en/fast/serve-modern-code-to-modern-browsers/index.md:122]:
```suggestion
### Enable Modern Bugfixes

`@babel/preset-env` groups multiple JavaScript syntax features into collections and enables/disables
them based on the target browsers specified. Although this works well, an entire collection of
syntax features is transformed when a targeted browser contains a bug with just a single feature.
This often results in more transformed code than is necessary.

Originally developed as a [separate preset](https://github.com/babel/preset-modules), the 
[bugfixes option](https://babeljs.io/docs/en/babel-preset-env#bugfixes) in `@babel/preset-env`
solves this problem by converting modern syntax that is broken in some browsers to the closest
equivalent syntax that is not broken in those browsers. The result is nearly identical modern code
with a few small syntax tweaks that guarantee compatibility in all target browsers. To use this
optimization, make sure you have `@babel/preset-env` 7.10 or later installed, then set the
[`bugfixes`](https://babeljs.io/docs/en/babel-preset-env#bugfixes) property to `true`:
```

File: src/site/content/en/fast/serve-modern-code-to-modern-browsers/index.md [hunk 3]
```diff
@@ -116,6 +108,35 @@ browsers to be used to access your application.
...
+     {
+       "bugfixes": true
+     }
+   ]
+ ]
+}
+```
+
+This feature is only supported in `@babel/preset-env` version 7.10 or later. Babel 8 will also
+include these optimizations by default.
```
Comment (developit) [inline src/site/content/en/fast/serve-modern-code-to-modern-browsers/index.md:138]:
```suggestion
This feature was introduced in `@babel/preset-env` version 7.10. In Babel 8, the
`bugfixes` option will be enabled by default.
```

File: src/site/content/en/fast/serve-modern-code-to-modern-browsers/index.md (hunk 1 above)
Comment (developit) [reply src/site/content/en/fast/serve-modern-code-to-modern-browsers/index.md:118]: I took a swing at trying to reword to address this, let me know what you think @housseindjirdeh.

**Comment by housseindjirdeh** [comment]
@developit Definitely sounds better now, thanks for updating :)

File: src/site/content/en/fast/serve-modern-code-to-modern-browsers/index.md [hunk 4]
```diff
@@ -116,6 +108,36 @@ browsers to be used to access your application.
 Ultimately, you should select the appropriate combination of queries to only
 target browsers that fit your needs.
 
+### Enable Modern Bugfixes
```
Comment (kaycebasques) [inline src/site/content/en/fast/serve-modern-code-to-modern-browsers/index.md:111]:
```suggestion
### Enable modern bugfixes
```

Sentence case is our convention

File: src/site/content/en/fast/serve-modern-code-to-modern-browsers/index.md [hunk 5]
```diff
@@ -116,6 +108,36 @@ browsers to be used to access your application.
...
+       "bugfixes": true
+     }
+   ]
+ ]
+}
+```
+
+In Babel 8, the `bugfixes` option will be enabled by default.
+
 ## Use &lt;script type=&quot;module&quot;&gt;
```
Comment (kaycebasques) [inline src/site/content/en/fast/serve-modern-code-to-modern-browsers/index.md:141]:
```suggestion
## Use `<script type="module">`
```
Drive-by fix
